swap_meet/vendor.py

```diff
@@ -1,2 +1,63 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory=inventory
+    def add(self, item): 
+        self.inventory.append(item)
+        return item
+    def remove(self, item): 
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+    def get_by_category(self,category):
+        cat_list=[]
+        for item in self.inventory:
+            if item.category==category:
+                cat_list.append(item)
+        return cat_list
+    def swap_items(self, Vendor, my_item, their_item):
+        if my_item in self.inventory and their_item in Vendor.inventory:
+            self.remove(my_item)
+            Vendor.add(my_item)
+            Vendor.remove(their_item)
+            self.add(their_item)
+            return True
+        else:
+            return False
+    def swap_first_item(self, Vendor):
```

Comment:
Same comment about passing `Vendor` as a parameter as above. You already have a function that will do this work for you! swap_items() can be called here and then you would pass in the first item of the vender and the other vendor as arguments
return self.swap_items(other, self.inventory[0], other.inventory[0])